input/fsh/SD_Genomics.fsh

```diff
@@ -92,7 +96,7 @@ Description:  "A specimen taken from a Patient. The profile includes extensions
     BodyLocationQualifier named locationQualifier 0..* and
     LateralityQualifier named lateralityQualifier 0..1
 // It would be nice to reuse the existing "condition-related" extension but it doesn't apply to Specimen (see Jira https://jira.hl7.org/projects/FHIR/issues/FHIR-31027) 
-* extension contains RelatedCondition named relatedCondition 0..1 MS
+* extension contains RelatedCondition named relatedCondition 0..1 MS  // should this be 0..*?
```

Comment:
More generally...why did we need to add RelatedCondition as an extension to Specimen? It's not in US Core and not specified in the Clinical Genomics Reporting IG?
Is there a specific use case or organization of an actual implementer which identified a need for this?

Reply:
That's a mistake. Retracted.